locust/main.py

```diff
@@ -115,6 +116,15 @@ def parse_options():
         help="Port that locust master should bind to. Only used when running with --master. Defaults to 5557. Note that Locust will also use this port + 1, so by default the master node will bind to 5557 and 5558."
     )
 
+    parser.add_option(
+        '--expect-slaves',
```

Comment:
Wouldn't the current implementation start close to immediately once the min slave count has been reached? If so, how would I go about to start a test with more slaves than min_slaves (other than just spawning them all at the same time and hoping that they would connect before the the poll check)?